### PR TITLE
add steering as blog approval

### PR DESCRIPTION
**Pull request by linsun**
fixes: https://github.com/istio/istio.io/issues/7535

Not sure how to do require 2 approvers for this `/content/en/blog` folder only.  I can configure the istio.io repo to require 2 approvals, but that is not really what we wanted.  @howardjohn do you know?

[ ] Configuration Infrastructure
[ x ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,6 +1,7 @@
 *                @istio/wg-docs-maintainers-infra
 /.spelling       @istio/wg-docs-maintainers-english
 /content/en/     @istio/wg-docs-maintainers-english
+/content/en/blog @istio/steering-committee-maintainers @istio/steering-committee-members
```
Comment (howardjohn) [inline CODEOWNERS:4]:
```suggestion
/content/en/blog @istio/steering-committee 
```

I think the ones mentioned here dont exist?

Comment (linsun) [reply CODEOWNERS:4]: Thanks, I was putting the structure incorrect, thought it can read the maintainers and members section under steering committee

**Comment by linsun** [comment]
Thanks @howardjohn for reviewing and confirming my guess.  Given the limitation, we'll ask @istio/steering-committee to approve only after a doc maintainer reviewed it for technical accuracy.

**Comment by frankbu** [comment]
@linsun without joint approval, we better make sure the the steering committee members NEVER approve a PR until it has approval from a doc maintainer (will that be clear vs. approved by just some other Istio member?) because the PR will merge as soon as they approve.

**Comment by ericvn** [comment]
Or would it be easier to not have a Doc maintainer not approve it until a steering committee member has approved it? The problem with this ordering is that Steering committee members wouldn't get a notification of the PR.

**Comment by frankbu** [comment]
> The problem with this ordering is that Steering committee members wouldn't get a notification of the PR.

Yeah, and the problem with the other order is that doc maintainers won't get notification of the PR. This really doesn't work well either way.

Maybe we should just leave it as is and we (doc maintainers) will request approval from the steering committee for any external or otherwise questionable blog PR, before we merge.

**Comment by craigbox** [comment]
For now, could you please tag Steering on all blog posts, so we're at least aware of them?

**Comment by linsun** [comment]
That works for me @frankbu.  I learned knative has a similar 2 phase process we may be able to borrow.  I think @macruzbar will provide that details for us.

**Comment by howardjohn** [comment]
They probably use prow OWNERs files which allows this, but CODEOWNERs is
less flexible

**Comment by frankbu** [comment]
@howardjohn Is there (or can we add) a reviewer label for steering committee members, that we can use to notify them of PRs that need their attention?

**Comment by howardjohn** [comment]
@frankbu I don't know if you can automatically, but you can probably add them as reviewers.. although trying to do it now looks like I can assign literally every team *but* steering. Maybe a permissions thing. You can do `@istio/steering-committee` though

**Comment by frankbu** [comment]
Closing this, we agreed to keep things as they are but from now on will tag all blog PRs with @istio/steering-committee.

**Pull request closed without merging**